### PR TITLE
Some proxy thing

### PR DESCRIPTION
**Pull request by pikesley**

### CODE REVIEW

File: lib/blocktrain/client.rb [hunk 1]
```diff
@@ -2,8 +2,12 @@ module Blocktrain
   class Client
 
     def self.results query, method = 'search'
-      r = Curl::Easy.http_post(endpoint(method), query.to_json)
-      JSON.parse r.body_str
+      # This seems redundant, but it keeps the tests happy :/
+      require "pry" ; binding.pry
```
Comment (pezholio) [inline lib/blocktrain/client.rb]: So prying in makes the tests pass?

**Comment by pezholio** [comment]
```
2016-01-13T16:57:13.076075+00:00 app[web.1]: 2016-01-13 16:57:13 - ArgumentError - comparison of Fixnum with nil failed:
2016-01-13T16:57:13.076079+00:00 app[web.1]:    /app/lib/blocktrain/paginated_query.rb:13:in `<'
```

*\* Something *\* is not working properly :/

**Comment by pezholio** [comment]
So, digging in, this seems to be Elasticsearch returning this error:

```
{"error"=>
  {"root_cause"=>[{"type"=>"parse_exception", "reason"=>"Failed to derive xcontent"}],
   "type"=>"search_phase_execution_exception",
   "reason"=>"all shards failed",
   "phase"=>"query_fetch",
   "grouped"=>true,
   "failed_shards"=>
    [{"shard"=>0,
      "index"=>"train_data",
      "node"=>"wfq_qKhXS4OC_uK9oiUMfg",
      "reason"=>{"type"=>"parse_exception", "reason"=>"Failed to derive xcontent"}}]},
 "status"=>400}
```

**Comment by pikesley** [comment]
Well that's baffling

**Comment by pezholio** [comment]
Totally, especially as the raw request works fine in Kopf

**Comment by pezholio** [comment]
This now works

File: lib/blocktrain/client.rb (hunk 1 above)
Comment (pikesley) [inline lib/blocktrain/client.rb]: Expert trolling

Comment (pezholio) [inline lib/blocktrain/client.rb]: :trollface:

**Pull request closed without merging**